front/app/components/ProjectAndFolderCards/ProjectAndFolderCardsInner.tsx

```diff
@@ -67,6 +69,7 @@ const ProjectAndFolderCardsInner = ({
   loadingInitial,
   loadingMore,
   hasMore,
+  currentlyWorkingOnText,
```

Comment:
The projects title text is now configurable via the homepage builder, as well.

Reply:
We will delete the app config option after the migration, for now we need it for backwards-compatibility.